### PR TITLE
Support Poll Delay

### PR DESCRIPTION
**Pull request by patricker**
Allow users to specify a polling delay (defaults to 100ms)

### CODE REVIEW

File: TaskClient.cs [hunk 1]
```diff
@@ -11,8 +11,8 @@ namespace Gofer.NET
     public class TaskClient
     {
         private static readonly object Locker = new object();
-        
-        private const int PollDelay = 100;
+
+        public int PollDelay { get; set; } = 100;
```
Comment (brthor) [inline TaskClient.cs]: Leaving a configuration value of the TaskClient in a settable property sets off my code spidey sense.

I'd prefer to have a pollDelay parameter (with a default value of 100) in the constructor that sets a private field _pollDelay.

**Comment by patricker** [comment]
Updated based on your feedback.

File: TaskClient.cs [hunk 2]
```diff
@@ -80,6 +82,7 @@ public CancellationTokenSource Start()
                     }
                     
                     await ExecuteQueuedTask();
+                    Thread.Sleep(_pollDelay);
```
Comment (brthor) [inline TaskClient.cs]: Use `await Task.Delay` for more efficient use of the thread pool.

Comment (patricker) [reply TaskClient.cs]: Thanks for the suggestion, I didn't know about Task.Delay.

**Comment by patricker** [comment]
Updated.

**Comment by brthor** [comment]
This looks good, I am going to re-enable the travis test runs to ensure none of the tests break before merging it though.

**Comment by patricker** [comment]
Looks like it succeeds on Linux but fails on OSX?
In my environment I'm developing on Windows and running on Linux.

Look at historic builds, it looks like the OSX unit tests just fail from time to time?

**Pull request closed without merging**